packages/code-du-travail-frontend/src/common/Breadcrumbs.js

```diff
@@ -2,28 +2,36 @@ import React from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 import { ChevronsRight } from "react-feather";
-import { List, ListItem, OverflowWrapper, theme } from "@socialgouv/react-ui";
+import {
+  Container,
+  List,
+  ListItem,
+  OverflowWrapper,
+  theme
+} from "@socialgouv/react-ui";
 
 const Breadcrumbs = ({ items = [] }) => {
   if (!items || items.length === 0) {
     return null;
   }
   return (
     <Nav>
-      <OverflowWrapper shadowColor="white">
-        <StyledList>
-          {items.map((item, index) => (
-            <React.Fragment key={index}>
-              <NavItem>{item}</NavItem>
-              {index < items.length - 1 && (
-                <Separator aria-hidden>
-                  <StyledChevronsRight />
-                </Separator>
-              )}
-            </React.Fragment>
-          ))}
-        </StyledList>
-      </OverflowWrapper>
+      <StyledContainer>
+        <OverflowWrapper shadowColor="white">
+          <StyledList>
+            {items.map((item, index) => (
+              <React.Fragment key={index}>
```

Comment:
Use index to key is an anti-pattern is better to use item Id or generate unique id

Reply:
I'm quite sure it's not an issue here